### DISCUSSION TITLE
Required workflows not triggering PR closed events anymore

### DISCUSSION THREAD

**Discussion by ajmalab** [community/community#63026] [Actions] [answered]
### Select Topic Area

Question

### Body

Hello, 

We have a  required workflow that's meant to run whenever a PR is closed and had been working fine until Thursday last week when it randomly stopped. From my tests, required workflows isn't receiving the `closed` event anymore. Could not find anything in the GH changelog so was wondering, was anything changed?

**Comment by Phoenix-Genius** [comment]
I'm sorry to hear that you're experiencing issues with your required workflow not triggering PR closed events anymore. While I don't have access to specific GitHub changelogs, it's possible that there may have been changes or updates to the GitHub Actions system that could have affected this behavior. I recommend checking the GitHub Actions documentation or reaching out to GitHub Support for more information and assistance in troubleshooting the issue. They should be able to provide you with more insights and help you resolve the problem.

**Comment by Phoenix-Genius** [comment]
Now, please give me star.
https://github.com/devfairy0830/nano-neuron

**Comment by Spixmacaw** [comment]
So as found in the comment below the post itself, if you want the workflow to run on branch x the .github folder must be on branch x and any other branch you want to trigger the workflow from

**Comment by ajmalab** [comment]
@Spixmacaw This is about Required Workflows, not the repo specific ones.

**Comment by ajmalab** [comment, marked as answer]
To anybody else facing this issue, got some answers from their support; seems Github silently changed this behaviour and silently updated the docs to reflect that. See documentation change PR [here](https://github.com/github/docs/commit/17e82b96e70ebe9c228a7c7bc06948425c2e0986).
